lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -17,12 +17,13 @@
                     this.courseID = options.courseID;
                     this.discussionTopicID = this.model.get('discussion_topic_id');
```

Comment:
Can you also remove this setting and just access it from the model like you are doing with memberships.